src/components/ContactsList/ContactsHeader.jsx

```diff
@@ -1,39 +1,28 @@
-import React, { Component } from "react";
-import { Button, IntentOpener } from "cozy-ui/react";
+import React from "react";
+import PropTypes from "prop-types";
 
-const ContactsFilter = () => <div>ContactsFilter</div>;
+const ContactsFilter = () => (
+  <div>
+    <select name="text">
+      <option value="value1">Value 1</option>
+      <option value="value2" selected>
+        Value 2
+      </option>
+      <option value="value3">Value 3</option>
+    </select>
+  </div>
+);
 
-const ContactsCreationButton = () => <div>ContactsCreationButton</div>;
-
-class ContactsIntentLink extends Component {
```

Comment:
I move it to https://github.com/cozy/cozy-contacts/pull/48/commits/12af99629749f7770df727c1248895c0b5fc3792#diff-c905dcf166d488b631fbf320ce845a53